src/msalexample-ios-macos/msalExample-macOS/MicrosoftLoginView.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+import MSAL
+
+struct MicrosoftLoginView: View {
+    @State private var isAuthenticated = false
+
+    var body: some View {
+        VStack {
+            Text("Welcome to the SwiftUI MSAL tutorial for macOS")
+            Button {
+                if (!isAuthenticated) {
+                    MSALAuthentication.Signin(MSALWebviewParameters(), completion: { (account, securityToken, error) in
+                        if let _ = error {
+                            return
+                        }
+
+                        isAuthenticated.toggle()
+                    })
+                }
+                else {
+                    // IMPORTANT: this require keychain capabilities to be added and signin with a development team. For more information, please refer
```

Comment:
"and signin with a development team"  -- is that supposed to be "and signed with a valid development certificate" instead?

Otherwise, I'm not understanding what the call to action is relative to the link you sent.

And to be clear, does only the signout flow need this in place?

Reply:
yup, that is correct. only signout required this capability. 

Is the following about ok?
```suggestion
                    // IMPORTANT: this require keychain capabilities to be added and signed with a valid development certificate. For more information, please refer
```

Reply:
yeah, looks good.